src/elements/Avatar/Avatar.tsx

```diff
@@ -24,7 +23,7 @@ export interface AvatarProps extends React.HTMLAttributes<HTMLDivElement> {
   /**
    * Style variant for the avatar.
    */
-  variant?: 'filled' | 'outline';
+  variant?: 'filled' | 'outline' | 'colored';
```

Comment:
This variant name could be better. Its not obvious whats different here.

Reply:
Can we call this like `primary` or maybe `secondary` or something? this isn't a common name we use.

Reply:
`primary` or `secondary` it's more about color

Reply:
I think still no. Its not obvious what this is and how its different from the others. Like `filled` could mean `colored` imo.

Reply:
We had `color` property - now it is deprecated and moved to the `variant`. Also, user can pass `colorOptions`. IDK what you don't like.

Reply:
it seems like `colored` is the same things as `filled`, but with a diff background color. Is that right?

if a user wanted the avatar filled, but with a diff background color than their globally styled `filled` variant, it could be simpler for them to just pass an altered AvatarTheme to the component `theme` prop

Reply:
> it seems like colored is the same things as filled, but with a diff background color. Is that right?
Yes

The reason was to make component similar to figma design without breaking the functionality. Let me rethink it